gax-httpjson/src/test/java/com/google/api/gax/httpjson/HttpRequestRunnableTest.java

```diff
@@ -168,6 +168,24 @@ public void testRequestUrl() throws IOException {
     Truth.assertThat(httpRequest.getUrl().toString()).isEqualTo(expectedUrl);
   }
 
+  @Test
+  public void testRequestUrlUnnormalized() throws IOException {
+    httpRequestRunnable =
+        HttpRequestRunnable.<CatMessage, EmptyMessage>newBuilder()
+            .setHttpJsonCallOptions(fakeCallOptions)
+            .setEndpoint("www.googleapis.com/animals/v1/projects")
+            .setRequest(catMessage)
+            .setApiMethodDescriptor(methodDescriptor)
+            .setHttpTransport(new MockHttpTransport())
+            .setJsonFactory(new JacksonFactory())
+            .setResponseFuture(SettableApiFuture.<EmptyMessage>create())
+            .build();
+    HttpRequest httpRequest = httpRequestRunnable.createHttpRequest();
+    Truth.assertThat(httpRequest.getContent()).isInstanceOf(EmptyContent.class);
+    String expectedUrl = ENDPOINT + "name/feline" + "?food=bird&food=mouse&size=small";
```

Comment:
nit: though I normally favor code reuse, consider NOT using `ENDPOINT` here and being explicit. That would make it more obvious to me as a reader that the protocol and the trailing endpoint `/` are being added.

Reply:
still a nit, but please do consider this ;-)

Reply:
Agreed. Please do this.